<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtUserSearchFirstPage</class>
 <widget class="QWizardPage" name="QtUserSearchFirstPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string></string>
  </property>
  <property name="title">
   <string>Add a user</string>
  </property>
  <property name="subTitle">
   <string>Add another user to your contact list. If you know their address you can add them directly, or you can search for them.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="howLabel_">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="byJID_">
       <property name="text">
        <string>I know their address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jid_"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="byLocalSearch_">
     <property name="text">
      <string>I'd like to search my server</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="byRemoteSearch_">
       <property name="text">
        <string>I'd like to search another server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="service_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>77</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel_">
     <property name="text">
      <string></string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
